tests/test_autodoc.py

```diff
@@ -889,6 +898,10 @@ def __special2__(self):
         # undocumented special method
         pass
 
+    def yodele(self):
```

Comment:
While I like funny method names as much as anybody else, could we pick some more descriptive names for this?  (Same for `yolo` below.)